### DISCUSSION TITLE
soil moisture measurement

### DISCUSSION THREAD

**Discussion by 4irm** [jeroenterheerdt/HAsmartirrigation#476] [Ideas]
Hi there,

I do have an underground irrigation which is for non lawn very common because you can safe up to 70% of water...there are several brands such as Gardena micro-drip, hunter, rainbrid etc.

Therefore I do have a question; does the irrigation calculation also works for that use case - or just lawn and beds?

Secondly - and maybe related to the question above - wouldn't make a soil moisture sensor make sense?

Thanks!

**Comment by jeroenterheerdt** [comment]
not sure I understand what is an underground irrigation system - do you have a picture or a link where I can read more before answering that question?

Soil moisture sensors don't make sense - see here why: https://github.com/jeroenterheerdt/HAsmartirrigation/issues/456#issuecomment-2297723521

**Reply by 4irm** [reply]
Thanks for your feedback...

Gardena Microdrip:

![image](https://github.com/user-attachments/assets/46e165d3-0451-46fe-842d-94b7edaed4b6)

So basically underground irrigation, together with a soil moisture sensor the right amount of water could be calculated.

Thanks!

**Reply by jeroenterheerdt** [reply]
You can apply the integration to this no problem. You can't use a soil moisture sensor with this integration and you don't need one.
